src/main/java/entity/Note.java

```diff
@@ -86,4 +86,9 @@ public LocalDateTime getDate() {
     public void setDate(LocalDateTime date) {
         this.date = date;
     }
+
+    @Override
+    public String toString() {
+        return this.getID() + " " + this.date + " " + this.text;
```

Comment:
I'm not sure we need to show the ID here, as it isn't relevant to the end user. Could we take the ID part out?

Reply:
sure